### DISCUSSION TITLE
Transparency not working Properly

### DISCUSSION THREAD

**Discussion by ManuelAGC** [mozilla/hubs#3517] [Spoke Questions] [answered]
I made a 3D model on blender that uses a texture sheet with alpha channel. The Model showed perfectly in blender with the material set to "alpha clip" as the blend mode on the material settings. The transparency is not rendered properly in Spoke and the results are of poor quality.

![image](https://user-images.githubusercontent.com/32258795/101638219-79bce680-3a36-11eb-9512-a800a006d52e.png)
![image](https://user-images.githubusercontent.com/32258795/101638240-7e819a80-3a36-11eb-8bc2-a4c0769f2571.png)

**Comment by ManuelAGC** [comment, marked as answer]
I found a compromise. Deleting all the lights in Spoke solves the problem, I guess it is related to the rendering order(?). Not being able to use any light source within spoke is not ideal, but in this particular case baked lights in blender are good enough for me.

**Reply by j-conrad** [reply]
Interesting... I hadn't considered lights being a factor here. Out of curiosity, how many lights were hitting the tree?

**Reply by ManuelAGC** [reply]
At first the scene had 4 lights: 3 point lights and a spotlight. The spotlight was right above the tree but I would say the tree was probably hit by all of them. I took them away and I replaced them by an ambient light so I could control the overall brightness but the problem persisted. Only when I disabled all the lights (not counting the skybox) the problem went away

**Reply by elmau** [reply]
Change the setting to alpha blend, it has worked for me.
